test/lib/code_corps/user_task_matcher_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule CodeCorps.UserTaskMatcherTest do
+  use CodeCorps.ModelCase
+
+  import CodeCorps.UserTaskMatcher
+
+  test "can find top x tasks for a user's skill" do
+    coding = insert(:skill, title: "coding")
+    design = insert(:skill, title: "design")
+
+    account_page = insert(:task)
+    settings_page = insert(:task)
+    photoshop = insert(:task)
```

Comment:
Yeah I think this is simply the most naive approach right now. I'd prefer some sort of recommender system generally but we're a ways off from that.